.github/workflows/build.yaml

```diff
@@ -12,30 +11,53 @@ on:
   release:
     types: [published]
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
   test:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
       matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
         emacs_version:
-          - "24.5"
-          - "25.3"
           - "26.3"
           - "27.2"
           - "28.2"
```

Comment:
Add 29 here?

Reply:
There is no 29 yet, it will be released in next year 2023 spring.

Reply:
There is no 29 yet, it will be released in next year 2023 spring.